test/cmdlineTests.sh

```diff
@@ -156,6 +156,18 @@ printTask "Testing passing empty remappings..."
 test_solc_file_input_failures "${0}" "=/some/remapping/target" "" "Invalid remapping: \"=/some/remapping/target\"."
 test_solc_file_input_failures "${0}" "ctx:=/some/remapping/target" "" "Invalid remapping: \"ctx:=/some/remapping/target\"."
 
+printTask "Testing passing location printing..."
+(
+cd "$REPO_ROOT"/test/errorReports/
```

Comment:
`errorReports` is a little unspecific inside the `test` directory. Perhaps `cmdlineErrorReports`? We could also move the whole cmdline test suite into its own subdirectory.

Reply:
sure